src/test/init/features/storage.spec.ts

```diff
@@ -43,11 +47,26 @@ describe("storage", () => {
         rcfile: {},
         projectId: "my-project-123",
       };
+      checkApiStub.returns(true);
 
       expect(doSetup(setup, new Config("/path/to/src", {}))).to.eventually.be.rejectedWith(
         FirebaseError,
         "Cloud resource location is not set"
       );
     });
+
+    it("should error when the storage API is not enabled", async () => {
+      const setup = {
+        config: {},
+        rcfile: {},
+        projectId: "my-project-123",
+      };
+      checkApiStub.returns(false);
+
+      expect(doSetup(setup, new Config("/path/to/src", {}))).to.eventually.be.rejectedWith(
+        FirebaseError,
+        "It looks like you haven't used Cloud Storage"
```

Comment:
Did you want to finish this messaging? or am I missing something?

Reply:
This is a test, it's just looking for a message prefix.